### PR TITLE
added properties files for READMEs for Thorntail

### PR DESCRIPTION
**Pull request by rhoads-zach**
@gastaldi @tradej let me know what you think

### CODE REVIEW

File: docs/topics/readme/cd-rest-http-thorntail.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (gastaldi) [inline docs/topics/readme/cd-rest-http-thorntail.properties]: The plugin is different for Thorntail. I believe it is now `thorntail:run` or similar @Ladicek or @kenfinnigan can confirm

File: docs/topics/readme/cd-cache-thorntail.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x.
+cicdSection=IMPORTANT: As part of the process of creating this booster, developers.redhat.com/launch or the Fabric8 Launcher set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster or OpenShift Online Web Console.
+gitCMD=git clone git@github.com:${loggedUser}/${targetRepository}
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+app-name=wfswarm-cache-greeting
```
Comment (gastaldi) [inline docs/topics/readme/cd-cache-thorntail.properties]: Not sure if the app name changes too /cc @Ladicek?

**Comment by tradej** [comment]
Will these properties files be visible anywhere? If so, I would wait for the name change to take effect first and only then merge this PR. Apart from @gastaldi's comments, I believe this looks good.

File: docs/topics/readme/cd-cache-thorntail.properties (hunk 2 above)
Comment (Ladicek) [reply docs/topics/readme/cd-cache-thorntail.properties]: Yea, `thorntail-cache-greeting`.

File: docs/topics/readme/cd-circuit-breaker-thorntail.properties [hunk 3]
```diff
@@ -0,0 +1,10 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/cd-circuit-breaker-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/cd-configmap-thorntail.properties [hunk 4]
```diff
@@ -0,0 +1,12 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/cd-configmap-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/cd-configmap-thorntail.properties [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+fileLocation=src/main/webapp/index.html
+OSOConfigMap=$ oc create configmap app-config --from-file=app-config.yml
+app-name=booster-configmap-wildfly-swarm
```
Comment (Ladicek) [inline docs/topics/readme/cd-configmap-thorntail.properties]: Not sure where this comes from, but likely needs changing.

File: docs/topics/readme/cd-crud-thorntail.properties [hunk 6]
```diff
@@ -0,0 +1,7 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/cd-crud-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/cd-health-check-thorntail.properties [hunk 7]
```diff
@@ -0,0 +1,9 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/cd-health-check-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/cd-health-check-thorntail.properties [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+fileLocation=src/main/webapp/index.html
+app-name=booster-health-check-wildfly-swarm
```
Comment (Ladicek) [inline docs/topics/readme/cd-health-check-thorntail.properties]: Again not sure where this comes from, but it likely needs changing.

File: docs/topics/readme/cd-rest-http-secured-thorntail.properties [hunk 9]
```diff
@@ -0,0 +1,6 @@
+localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
+app-name=wf
```
Comment (Ladicek) [inline docs/topics/readme/cd-rest-http-secured-thorntail.properties]: ??

File: docs/topics/readme/cd-rest-http-thorntail.properties (hunk 1 above)
Comment (Ladicek) [reply docs/topics/readme/cd-rest-http-thorntail.properties]: Correct, `mvn thorntail:run`.

File: docs/topics/readme/zip-cache-thorntail.properties [hunk 10]
```diff
@@ -0,0 +1,8 @@
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x.
+cicdSection=
+gitCMD=unzip ${artifactId}.zip
+targetRepository=${artifactId}
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+app-name=wfswarm-cache-greeting
```
Comment (Ladicek) [inline docs/topics/readme/zip-cache-thorntail.properties]: `thorntail-cache-greeting`

File: docs/topics/readme/zip-circuit-breaker-thorntail.properties [hunk 11]
```diff
@@ -0,0 +1,11 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/zip-circuit-breaker-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/zip-configmap-thorntail.properties [hunk 12]
```diff
@@ -0,0 +1,13 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/zip-configmap-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/zip-configmap-thorntail.properties [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+fileLocation=src/main/webapp/index.html
+OSOConfigMap=$ oc create configmap app-config --from-file=app-config.yml
+app-name=booster-configmap-wildfly-swarm
```
Comment (Ladicek) [inline docs/topics/readme/zip-configmap-thorntail.properties]: Again not sure where this comes from, but it likely needs changing.

File: docs/topics/readme/zip-crud-thorntail.properties [hunk 14]
```diff
@@ -0,0 +1,7 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/zip-crud-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/zip-health-check-thorntail.properties [hunk 15]
```diff
@@ -0,0 +1,10 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/zip-health-check-thorntail.properties]: `mvn thorntail:run`

File: docs/topics/readme/zip-health-check-thorntail.properties [hunk 16]
```diff
@@ -0,0 +1,10 @@
...
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+fileLocation=src/main/webapp/index.html
+app-name=booster-health-check-wildfly-swarm
```
Comment (Ladicek) [inline docs/topics/readme/zip-health-check-thorntail.properties]: Again not sure where this comes from, but it likely needs changing.

File: docs/topics/readme/zip-rest-http-secured-thorntail.properties [hunk 17]
```diff
@@ -0,0 +1,6 @@
+localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
+app-name=wf
```
Comment (Ladicek) [inline docs/topics/readme/zip-rest-http-secured-thorntail.properties]: ??

File: docs/topics/readme/zip-rest-http-thorntail.properties [hunk 18]
```diff
@@ -0,0 +1,9 @@
+localRunCMD=mvn wildfly-swarm:run
```
Comment (Ladicek) [inline docs/topics/readme/zip-rest-http-thorntail.properties]: `mvn thorntail:run`

**Comment by Ladicek** [comment]
> Will these properties files be visible anywhere? If so, I would wait for the name change to take effect first and only then merge this PR.

We (I, actually) submitted a PR to the booster catalog yesterday that moves WildFly Swarm community boosters to Thorntail. The PR was merged, so Launcher already generates Thorntail projects. So these properties are probably urgently needed :-)

**Comment by tradej** [comment]
@Ladicek, I see. Consider my comment withdrawn.

**Comment by rhoads-zach** [comment]
@gastaldi @Ladicek @tradej updated per feedback

File: docs/topics/readme/cd-rest-http-secured-thorntail.properties [hunk 19]
```diff
@@ -0,0 +1,6 @@
+localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
+app-name=thorntail
```
Comment (Ladicek) [inline docs/topics/readme/cd-rest-http-secured-thorntail.properties]: ??

File: docs/topics/readme/zip-rest-http-secured-thorntail.properties [hunk 20]
```diff
@@ -0,0 +1,6 @@
+localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
+app-name=thorntail
```
Comment (Ladicek) [inline docs/topics/readme/zip-rest-http-secured-thorntail.properties]: Hmm, it probably used to be like that...

File: docs/topics/readme/cd-rest-http-secured-thorntail.properties (hunk 19 above)
Comment (rhoads-zach) [reply docs/topics/readme/cd-rest-http-secured-thorntail.properties]: It's used here: https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/readme/rest-http-secured-README.adoc#interacting-with-the-booster-on-a-single-node-openshift-cluster

**Comment by rhoads-zach** [comment]
thank you @Ladicek and @gastaldi merging.

**Pull request merged**